### DISCUSSION TITLE
Importing data with date

### DISCUSSION THREAD

**Discussion by gpjorgenson** [Budibase/budibase#5719] [Help] [answered]
I have been trying to import existing data into a postgresql table I created in budibase. I have tried exporting test data from the budibase table in the postgresql database where I manually put in the data. After exporting the data, budibase will not import the data it just exported. It just "Unable to import data" unless I remove the data/time columns. I have tried importing data using dbeaver and,once again, that works as long as I remove the date fields. I have tried every syntax I could think of from '2020-09-24T13:12:04.622Z' to  '2020-03-12 12:00:00' even '2021-10-12' and '2021-10-20 12:00:00:000 -0600' ... with every combination of single quotes, double quotes, double/single quotes (like budibase's export) and even no quotes... In budibase import, it gives the error I previously mentioned. In dbeaver, they all come up with: SQL Error [22007]: ERROR: invalid input syntax for type timestamp with time zone: ""

We have picked budibase as our low code selection but if I cannot import the existing dates along with the rest of the data, I am in trouble.

I am sure I can convert the dates into different formats for upload but the ISO8601 format does not import with either budibase's own import or another tool.

Can someone help me find a date format that will import?

**Comment by melohagan** [comment, marked as answer]
Hi @gpjorgenson 

This looks to be a bug. I noticed that the export includes an additional set of single quotes wrapped around the timestamps. I removed these, and the import was successful.

I've raised a bug here: https://github.com/Budibase/budibase/issues/5781

**Reply by gpjorgenson** [reply]
Can't wait to try it, thanks!

**Reply by gpjorgenson** [reply]
Removing the interior quotes did work. Thank you. I look forward to the bug getting fixed.
